SmartHouseGatewayApp/SmartHouseDataStore/Entities/Models/Condition.cs

```diff
@@ -0,0 +1,31 @@
+﻿using SmartHouseAbstraction.DataStore.Model;
+using System.ComponentModel.DataAnnotations;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SmartHouseDataStore.Entities
+{
+    public class Condition:IEntity
```

Comment:
Add space After Condition And before IEntity, on all classes.
Something like `Condition : IEntity`